content/blog/2022/05/verifying-image-signatures.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Verifying image signatures
+authors:
+- Raul Cabello Martin
+date: 2022-05-02
+---
+
+We continue to add support for verifying the integrity and authenticity of artefacts within Kubewarden using the Sigstore project. In this
```

Comment:
I would prefer something that makes it sound more production ready, and also points to previous blogpost:

> After this last releases Kubewarden now has support for verifying the integrity and authenticity of artefacts in the clusterf using the Sigstore project. In this post, we shall focus on verifying container image signatures using the
new [verify-image-signatures](https://github.com/kubewarden/verify-image-signatures) policy.
> 
> If you don't know how Sigstore works, our want to read on how to verify the authenticity of Kubewarden policies in particular, we recommend reading our previous post.

Reply:
previous blog is mentioned [here](https://github.com/kubewarden/kubewarden.io/pull/87/files#diff-cc14700ddc3ca1c4b8a11b8e6bcec540ac70a39579a1214ce141e8765d5a1d93R13)
How would you make it some more production ready?

Reply:
I suppose I was thinking about the first words, from "we continue to add support", to "After these last releases Kubewarden now has support".

Reply:
done in https://github.com/kubewarden/kubewarden.io/pull/87/commits/6b6e154fd50b994a15592acf04c331a982d8ab6a